### PR TITLE
Add a transition tool

### PR DESCRIPTION
**Pull request by qjcXu**
- Upload plugins.json

### CODE REVIEW

**Comment by qjcXu** [comment]
Could you tell me where I went wrong

File: plugins.json [hunk 1]
```diff
@@ -908,5 +908,19 @@
...
+	  "Name":"Translate Tool",
+	  "Description": "a plugin of transition",
+	  "Author": "qjc",
+	  "Version": "0.0.1",
+	  "Language": "executable",
+	  "Website": "https://github.com/qjcXu/Flow.Launcher.Transition",
+	  "UrlDownload": "https://github.com/qjcXu/Flow.Launcher.Transition/releases/download/v0.0.1/Flow.Launcher.Transition.zip"
+	  "UrlSourceCode": "https://github.com/qjcXu/Flow.Launcher.Transition"
+	  "IcoPath": "https://cdn.jsdelivr.net/gh/qjcXu/Flow.Launcher.Transition/src/assets/imgs/translate.png",
+	  "ExecuteFileName": "star.bat"
```
Comment (taooceros) [inline plugins.json:924]: you don't need the executefilename here, and the version, and the actionkeyword

**Comment by taooceros** [comment]
@Garulf could you take a look on the json validator?

File: plugins.json (hunk 1 above)
Comment (qjcXu) [reply plugins.json:924]: I'm sorry so I need to remove executefilename?

Comment (taooceros) [reply plugins.json:924]: Yeah please. Thanks

Comment (taooceros) [reply plugins.json:924]: Sure

**Comment by Garulf** [comment]
> @Garulf could you take a look on the json validator?

What do you mean?

**Pull request closed without merging**